apps/leo_gateway/priv/leo_gateway.conf

```diff
@@ -109,6 +109,16 @@ bucket_prop_sync_interval = 300
 ## wtperf is set to the same value.
 ## nfs.nfsd.wtmax = 5242880
 
+## NFSd's Readdir Entry Table Scan Interval in seconds (leo-project/leofs#612)
+## nfs.nfsd.readdir_scan_int = 1800
+
+## NFSd's Readdir Entry TTL in seconds (leo-project/leofs#612)
+## nfs.nfsd.readdir_entry_ttl = 3600
+
+## NFSd's Readdir Entry Table Memory Scanning Threshold in bytes (leo-project/leofs#612)
+## nfs.nfsd.readdir_mem_thres = 268435456
+
```

Comment:
Here is the result for 1000 files

## Generation
parallel "cp testfile_1mb /mnt/leofs/test/testfile{}" ::: $(seq 0 1000)

## ETS Memory Usage
** Initial **
```
ets:info(readdir_entry_ets_tbl).
[{compressed,false},
 {memory,299},
 {size,0}]
```

** ls /mnt/leofs/test **
```
ets:info(readdir_entry_ets_tbl).
[{compressed,false},
{memory,70667},
{size,2}]
```

** ls /mnt/leofs/test (once again) **
```
ets:info(readdir_entry_ets_tbl).
[{compressed,false},
{memory,141035},
{size,4}]
```